pypit/artrace.py

```diff
@@ -1485,6 +1485,11 @@ def pad_dict(indict):
             badlines += 1
             trcdict = pad_dict(trcdict)
             continue
+        if j == 74:  # KLUDGE!!!!
```

Comment:
Yikes!   :-)  Is there a reasonable reason for this, or can it now be removed if you were just testing?